quiche/src/recovery/gcongestion/recovery.rs

```diff
@@ -354,12 +357,26 @@ impl RecoveryEpoch {
 struct LossThreshold {
     pkt_thresh: u64,
```

Comment:
Should the type of pkt_thresh be changed to `Option<u64>` in order to avoid DISABLE_PKT_THRESHOLD?

Reply:
Yea considered that to make it more obvious. That might be cleaner than the `DISABLE_PKT_THRESHOLD`. Let me explore what that looks like.